xcdat/dataset.py

```diff
@@ -304,7 +305,13 @@ def decode_non_cf_time(dataset: xr.Dataset) -> xr.Dataset:
     time = ds.cf["T"]
     time_bounds = ds.get(time.attrs.get("bounds"), None)
     units_attr = time.attrs.get("units")
-    units, ref_date = _split_time_units_attr(units_attr)
+    # if the time axis cannot be split, we do not yet
+    # support time decoding and we return the original
+    # dataset
+    try:
+        units, ref_date = _split_time_units_attr(units_attr)
+    except ValueError:
+        return ds
```

Comment:
If there is no "since" in the time units, then the dataset is non-cf-compliant and unsupported, so instead of continuing this function (`decode_non_cf_time`) simply returns the original (non-decoded) dataset.